deps/wrapper.cc

```diff
@@ -131,6 +183,44 @@ bool JuliaGcsClient::Exists(const std::string &ns,
     return exists;
 }
 
+ObjectID _submit_task(std::string project_dir) {
+    auto &worker = CoreWorkerProcess::GetCoreWorker();
+
+    RayFunction func(
+        Language::JULIA,
+        FunctionDescriptorBuilder::BuildJulia("", "demo_task", "")
+    );
```

Comment:
this is just here to get the raylet to schedule the task on a julia worker right?

Reply:
Correct. In the next iteration of this the function name here will be used to call a specific Julia function.